Quick updates about credit cards and CS&S accounting (#743)
2i2c-org/team-compass: finance/payments.md

Used only for paying cloud providers (GCP, AWS, and Azure) for cloud costs.
We generally intend to re-imburse ourselves via invoices to communities for any cost charged to this credit card.

### Extending card spending limits

> We have a monthly spending limit on each card, and credit card purchases will begin being declined if we hit it.

Author: ```suggestion
We have a monthly spending limit on each card, and credit card purchases will be declined if we hit it.
```
